blazor/rich-text-editor/resizable-editor.md

```diff
@@ -19,7 +19,7 @@ The Rich Text Editor allows it to be resized dynamically, so the users can able
 {% endhighlight %}
 {% endtabs %}
 
-![Resizing in Blazor RichTextEditor](./images/blazor-richtexteditor-resizing.png)
+![Blazor RichTextEditor resizing](./images/blazor-richtexteditor-resizing.png)
 
 ## Restrict resize
 
```

Comment:
Highlighted the following words -  `min-width`, `max-width`, `min-height`, and `max-heigh`